azure-pipelines.yml

```diff
@@ -380,6 +374,89 @@ jobs:
       docker run $IMAGE_NAME /bin/bash -c "time az self-test && time az --version && sleep 5"
     displayName: 'Bash Script'
 
+- job: BuildAzureDockerImage
+  displayName: Build Azure Docker Image
+  dependsOn: BuildRpmPackageMariner
+  strategy:
+    matrix:
+      ${{ each arch in parameters.architectures }}:
+        Mariner 2.0 ${{ arch.name }}:
+          pool: ${{ arch.pool }}
+          artifactName: docker-mariner2.0-${{ arch.value }}
```

Comment:
The artifact name is `docker-mariner2.0-amd64` (align with the RPM artifact `rpm-mariner2.0-amd64`), the docker image tag is `cbl-mariner2.0`.

This is the reply from Mariner team:
> To keep things consistent with other images in MCR, can we make the tag "cbl-mariner2.0"? Once we release Mariner 3.0 next year, there will be a larger rebranding where we will be changing all container image tags to use "Azure Linux". However, for now we want to stay consistent with what already exists